playbooks/aws/openshift-node/terminate.yml

```diff
@@ -0,0 +1 @@
+../openshift-master/terminate.yml
```

Comment:
@jwhonce ah, the diff is a bit misleading, it is actually a symlink.  In reality, I could probably do something like openshift-cluster/launch.yml and openshift-cluster/launch-instances.yml, I was just trying to replicate the gce playbooks as much as possible.